examples/kernel_approximation/plot_scalable_poly_kernels.py

```diff
@@ -95,11 +94,14 @@
 # polynomial kernel of degree four would have approximately 8.5 million
 # features (precisely, 54^4). Thanks to :class:`PolynomialCountSketch`, we can
 # condense most of the discriminative information of that feature space into a
-# much more compact representation. We repeat the experiment 5 times to
-# compensate for the stochastic nature of :class:`PolynomialCountSketch`.
+# much more compact representation. While we run the experiment only a single time
+# (`n_runs` = 1) in this example, in practice one should repeat the experiment several
+# times to compensate for the stochastic nature of :class:`PolynomialCountSketch`.
+
+n_runs = 1
```

Comment:
I think it'd be nicer if you just remove the whole loop instead of running the loop once and just have the content of the loop in this example.

Reply:
This is a fair point and would make the code more concise; however, it does come at the cost of making repeat experimental runs (a practice encouraged in the documentation) non-trivial in the sense that it is less clear *at first glance* how to best repeat the experiment. 

The user would now have to think about where to place the internal loop and adjust both the `ps_lsvm_time` and `ps_lsvm_score` for the number of runs which, while not terribly difficult, certainly requires more effort than simply altering the value of `n_runs`.

If you believe this is a worthwhile tradeoff, I am happy to make the change you suggest here and update the PR. Otherwise, might I suggest keeping the loop to facilitate the user experience in this arguably minor way?

Reply:
I agree that keeping it makes it easier to a user to run the example with several runs to see how it improves the results. I'd be happy to keep it

Reply:
I am also fine keeping the for loop with the above comment.